[ENH] Adds sub_labels argument to first_level_from_bids (#3351)
nilearn/nilearn: doc/changes/latest.rst

Enhancements
------------

> - ``nilearn.glm.first_level.first_level.first_level_from_bids`` now takes an optional `subject_labels` argument and warns users of given subject labels that are not present in the dataset.

Reviewer: ```suggestion
- :func:`~glm.first_level.first_level_from_bids` now takes an optional ``sub_labels`` argument and warns users of given subject labels that are not present in the dataset (:gh:`3351` by `Kevin Sitek`_).
```
